code-studio/src/js/components/AdvancedShareOptions.jsx

```diff
@@ -62,6 +68,26 @@ var AdvancedShareOptions = React.createClass({
     );
   },
 
+  renderEmbedTab() {
+    var url = window.location.href.replace('edit','embed');
+    var iframeHtml = '<iframe width="352" height="612" style="border: 1px solid black;" src="' +
+          url + '"></iframe>';
```

Comment:
Why this particular width and height?  Can we derive those from any existing constants so that they don't get out of sync?